### ISSUE TITLE
Compatibility With Ears

### ISSUE THREAD

**Issue by halotroop2288** [js6pak/mojangfix#5]
Please disable the skin fix if [Ears](https://github.com/unascribed/ears) is present, for compatibility.

(Or simply set the Mixin's `required` value to `false`.)

**Comment by js6pak** [comment]
Seems like ears also fixes slim skin model so fixing compatibility probably wont be as simple as setting required to false

**Comment by halotroop2288** [comment]
That is true. I have tested this and it doesn't use slim models with my suggested fix.

**Comment by halotroop2288** [comment]
That would still break compatibility. Ears does more than just fix the skin. It adds more features to the skin renderer.

**Comment by telvarost** [comment]
I've created a compatibility fork of MojangFix utilizing GCAPI, allowing users to toggle off mixins that conflict with other mods.
You can use this fork if you'd like and simply have users disable skin fixes in the config. (Or, you can try to detect the config in your mod parse the json and force the config off), since it is a compat fork, I could also just add your mod to skin fixes mixin check as I did for my own Inventory tweaks mod.

Find it here: https://github.com/telvarost/MojangFix-StationAPI
Open to discussion on this. Not trying to create drama by making a fork. Just think its helpful for those looking for a slightly altered variant of MojangFix.
